expressions/aggregation/AggregationHandle.hpp

```diff
@@ -347,20 +348,15 @@ class AggregationHandle {
    */
   virtual void aggregateOnDistinctifyHashTableForGroupBy(
       const AggregationStateHashTableBase &distinctify_hash_table,
-      AggregationStateHashTableBase *aggregation_hash_table) const = 0;
+      AggregationStateHashTableBase *aggregation_hash_table,
+      int index) const = 0;
```

Comment:
Can you please update the doxygen for the ``index`` parameter? Also, if the index is always positive, can you change its type from int to std::size_t?